File: Java/Loops/DoWhile.java
public class DoWhile {
  public static void main(String[] args) {
    Boolean maybe = true;
    int i = 0;

    do {
      System.out.println(i);

      if (i == 9) {
        maybe = false;
      }

      i++;
    }
    while (maybe);
    
    System.out.println("Done.");
  }
}

File: C/Loops/do_while.c
#include <stdio.h>
#include <stdbool.h>

int main() {
  bool maybe = true;
  int i = 0;

  do {
    printf("%d\n", i);

    if (i == 9) {
      maybe = false;
    }

    i++;
  }
  while (maybe == true);

  printf("Done.");

  return 0;
}
